server/routerlicious/Dockerfile

```diff
@@ -3,7 +3,7 @@
 
 # DisableDockerDetector "No feasible secure solution for OSS repos yet"
 
-FROM node:16.19.1-buster-slim AS base
+FROM node:18.17.1 AS base
```

Comment:
You switched these (historian and R11s) from "-slim" to not-slim. Please add the -slim back. It's important for keeping the image size down.

Reply:
Changed to `18.17.1-buster-slim` in all dockerfiles.